package.json

```diff
@@ -27,73 +26,40 @@
     "linkifyjs": "^2.1.8",
     "object-hash": "^1.3.1",
     "reconnecting-websocket": "^3.2.2",
-    "vue": "~2.5.22",
+    "vue": "^2.6.10",
     "vue-flatpickr-component": "^7.0.6",
     "vue-h-ajax": "^0.1.3",
     "vue-i18n": "^8.11.1",
-    "vue-router": "^3.0.6",
+    "vue-router": "^3.0.3",
```

Comment:
Downgraded the router?

Reply:
Vue-cli has a mind of it's own. I'll manually upgrade it.